swagger.yml

```diff
@@ -3548,31 +3547,80 @@ paths:
             application/json:
               schema:
                 type: object
-                required:
-                  - Features
-                  - Technologies
-                  - Alerts
                 properties:
                   Features:
-                    $ref: '#/components/schemas/Features' 
-                  Alerts: 
-                    $ref: '#/components/schemas/AggregatedCodeSeverityAlerts'
-                  Technologies:
+                    $ref: '#/components/schemas/Features'
+                  technologies:
                     type: object
-                    required:
-                      - Technologies
-                      - Total
+                    description: Available technologies
                     properties:
+                      technologies:
+                        type: array
+                        items:
+                          type: object
+                          properties:
+                            product:
+                              type: string
+                              description: technologial product code
+                            Compliance:
+                              type: number
+                              description: |-
+                                a metric ranging [0..1] calculated as follows:
+
+                                if a ULA exists for a given product, its compliance := 1. When calculating the total, this product's purchasedLicenses := usedLicenses
+                                else if usedLicenses <= purchasedLicenses the product's compliance := 1. When calculating the total, this product's purchasedLicenses := usedLicenses
+                                else this product's compliance := purchasedLicenses / usedLicenses.
+                            unpaidDues:
+                              type: number
+                              description: |-
+                                "unpaidDues" = difference between calculated cost of used and purchased licenses:
+
+                                if usedLicensesCost <= purchasedLicensesCost, then unpaidDues := 0
+                                else unpaidDues := usedLicensesCost - purchasedLicensesCost
+                            hostsCount:
+                              type: number
+                              description: number of hosts found supporting the current product / total hosts found
+                      total:
+                        type: object
+                        properties:
+                          compliance:
+                            type: number
+                            description: sum(purchasedLicenses) / sum(usedLicenses) fore each product
+                          hostsCount:
+                            type: number
+                            description: sum of hostsCount
+                          unpaidDues:
+                            type: number
+                            description: sum(unpaidDues)
```

Comment:
Why do you chose to rewrite the Technologies object instead of modifying #/components/schemas/Features?

Reply:
the object Technologies it is an object appart and doesn't belong to Features. I don't know were Features is used and I don't want to compromise the code.

Reply:
ok updated the schema with the 2 ref's above.